skins/laika/src/view_models/DataPersistence.ts

```diff
@@ -0,0 +1,19 @@
+export enum DataPersistenceMutationType {
+	VALUE,
+	KEY_VALUE_PAIR,
+}
+
+export interface DataPersister {
+	save( key: string, data: string ): void;
+	load( key: string ): Promise<null | string>;
+	getPlugin( items: DataPersistenceItem[] ): any;
+	decryptInitialValues( items: DataPersistenceItem[] ): Promise<void>;
+	getInitialValue( key: string ): string|null;
+}
+
+export interface DataPersistenceItem {
```

Comment:
I guess this interface could also get a docblock description that explains what it is for and what its properties mean.